### PR TITLE
Add Travis build URL

### PR DESCRIPTION
**Pull request by andrewda**
This will only render the Travis build URL if the required
environment variables are set.

Closes https://github.com/coala/community/issues/8

### CODE REVIEW

**Comment by nkprince007** [comment]
ack https://github.com/coala/community/pull/30/commits/1a1a908bf1b7ddf1721a13e8d1fc1b4e083662db

File: gci/views.py [hunk 1]
```diff
@@ -48,6 +49,15 @@ def index(request):
              .format(unix=timegm(datetime.utcnow().utctimetuple()),
                      timestamp=timestamp))
 
+    repo_slug = os.environ.get('TRAVIS_REPO_SLUG')
```
Comment (jayvdb) [inline gci/views.py]: could you move the logic to a function so it is re-usable, and we can move the placement around.  separation of data and presentation.

(or maybe even create a django-travis pypi package which makes this information available.

c.f. https://github.com/jayvdb/jekyll-netlify ; we could create a django-netlify also ;-) )

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/30/commits/1a1a908bf1b7ddf1721a13e8d1fc1b4e083662db

File: gci/views.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 from django.http import HttpResponse
 from datetime import datetime
 from calendar import timegm
+from trav import Travis
 import requests
+import os
```
Comment (jayvdb) [inline gci/views.py]: whaa ... pep8 ... `os` belongs above third party libraries.

the current imports are a mess. we'll start to fix them without being disruptive.

could you put `os` under `calendar`, followed by a blank line, then `requests` then `trav`

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/30/commits/2d767b0c9d810bfdda5aaa70bb8b0e3b6e382f65

File: gci/views.py (hunk 2 above)
Comment (andrewda) [reply gci/views.py]: :+1:

Comment (andrewda) [reply gci/views.py]: Wait we don't even need `os`... 1 sec

**Comment by Monal5031** [comment]
@andrewda You have merge conflicts please fix them and make sure to do a rebase.

**Comment by hemangsk** [comment]
ack 82b3ec6

**Comment by nkprince007** [comment]
@gitmate-bot rebase this PR please.

**Comment by nkprince007** [comment]
ack https://github.com/coala/community/pull/30/commits/f8f22d50d4ef7742e01fdadf608b54e3bf33eef4

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/30/commits/f8f22d50d4ef7742e01fdadf608b54e3bf33eef4

**Comment by jayvdb** [comment]
@andrewda , ping

**Comment by jayvdb** [comment]
@andrewda `Distribution "python_dateutil" is not installed` is because it should be a hyphen.  edit .moban.yaml (check for any other `_` in the dependencies) , run `moban`, add new commit.

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/30/commits/98031ce058b958c1ae94fe2ae2b4d3f58c40b0d6 https://github.com/coala/community/pull/30/commits/ecd38b5e81ebaa18eba33eb7bd320cbf953608f8 https://github.com/coala/community/pull/30/commits/02db06a8d37e7865fe66677f1a5d712dac118be5

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**